client/views/main.js

```diff
@@ -163,6 +191,14 @@ module.exports = View.extend({
     stamp += (minutes < 10 ? '0' + minutes : minutes) + ":";
     return log.replace(time, stamp);
   },
+  getExampleLibrary: function () {
```

Comment:
A little confusing that the example library code is in this pr, but not a big deal